test/e2e/main_test.go

```diff
@@ -120,6 +134,25 @@ func TestMain(m *testing.M) {
 			}
 		}
 
+		var kbsparams string
+		if shouldDeployKbs {
+			log.Info("Deploying kbs")
+			if keyBrokerService, err = pv.NewKeyBrokerService(props["CLUSTER_NAME"]); err != nil {
+				return ctx, err
+			}
+
+			if err = keyBrokerService.Deploy(ctx, cfg, props); err != nil {
```

Comment:
The deployment failed for me as:

```bash
✗ ./caa-provisioner-cli -action=provision
INFO[0000] Creating VPC...
...
KBS Service IP: 10.0.241.140
INFO[0138] KBS PARAMS: "cc_kbc::http:10.0.241.140:8080":
FATA[0139] loading KV pairs: env source files: [service-principal.env]: evalsymlink failure on '/Users/suraj/temp/2024-02-Feb-20-19-09-13/cloud-api-adaptor/install/overlays/azure/service-principal.env' : lstat /Users/suraj/temp/2024-02-Feb-20-19-09-13/cloud-api-adaptor/install/overlays/azure/service-principal.env: no such file or directory
```

I had to run `✗ touch /Users/suraj/temp/2024-02-Feb-20-19-09-13/cloud-api-adaptor/install/overlays/azure/service-principal.env` and `✗ touch /Users/suraj/temp/2024-02-Feb-20-19-09-13/cloud-api-adaptor/install/overlays/azure/id_rsa.pub` to move further.